.github/ISSUE_TEMPLATE/documentation-request.md

```diff
@@ -2,7 +2,7 @@
 name: Documentation request
 about: Need help using the SDK? Spot an error? Let us know how our documentation can be improved.
 title: ''
-labels: documentation
```

Comment:
Should we keep the "documentation" labels as this is a documentation-request template? Sorry missed this in initial review.